### DISCUSSION TITLE
Welcome to adrenaline Discussions!

### DISCUSSION THREAD

**Discussion by shobrook** [shobrook/adrenaline#63] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lmsutools** [comment]
Congratulations for such amazing project. Eager to see what's next to come for this Shobrook. Please also try to find time to process some opened issues that people have opened in the last weeks.
